.github/workflows/ci.yml

```diff
@@ -62,20 +69,16 @@ jobs:
     - name: Set Rouge version
       if: matrix.rouge-version
       run: echo 'ROUGE_VERSION=${{ matrix.rouge-version }}' >> $GITHUB_ENV
-    - name: Install libxslt for Nokogiri (Linux)
-      if: matrix.os == 'ubuntu-latest'
-      run: sudo apt-get install libxslt1-dev
-    - name: Install Ruby
+    - name: Install Ruby (uses cached dependencies for non-scheduled build)
       uses: ruby/setup-ruby@v1
       with:
         ruby-version: ${{ matrix.ruby }}
         bundler-cache: ${{ github.event_name != 'schedule' }}
-    - name: Install dependencies without cache
+    - name: Install dependencies (scheduled build only)
       if: github.event_name == 'schedule'
-      shell: bash
       run: |
-        bundle config --local path vendor/bundle
-        bundle install --jobs 4
+        bundle config --local path .bundle/gems
+        bundle --jobs 3 --retry 3
```

Comment:
Err... `install` got lost?

Reply:
install is the default command, so there's not need to specify it. I tend not to use it in the CI jobs in the Asciidoctor ecosystem, so I was just being consistent with what I've done elsewhere.

Reply:
> install is the default command

I see, didn't know that.